traitsui/wx/tree_editor.py

```diff
@@ -1650,18 +1650,12 @@ def can_add_to_menu(self, action):
         """ Returns whether the action should be defined in the user interface.
         """
         if action.defined_when != '':
-            try:
-                if not eval(action.defined_when, globals(), self._context):
-                    return False
-            except:
-                open_fbi()
+            if not eval(action.defined_when, globals(), self._context):
```

Comment:
Same comment here: this code now matches what's done for Qt.

Reply:
This is definitely for the better, since `defined_when` is almost certainly user-defined code and so we've been silently ignoring issues with this.